packages/babel-core/src/config/caching.js

```diff
@@ -31,7 +31,7 @@ export function makeStrongCache<ArgT, ResultT>(
  * configures its caching behavior. Cached values are stored weakly and the function argument must be
  * an object type.
  */
-export function makeWeakCache<ArgT: {} | Array<*>, ResultT>(
+export function makeWeakCache<ArgT: {} | Array<*> | $ReadOnlyArray<*>, ResultT>(
```

Comment:
Doesn't the `ArgT: {}` already imply any object at all (including arrays)? Or is flow stricter about that than typescript?

Reply:
It didn't seem to. I believe I added this to fix errors.